sdks/python/apache_beam/io/fileio.py

```diff
@@ -708,6 +708,12 @@ def process(self, element, w=beam.DoFn.WindowParam):
         self.path.get(),
         move_to)
 
+    try:
+      filesystems.FileSystems.mkdirs(self.path.get())
+    except IOError:
```

Comment:
BeamIOError? can we log this as debug?

Reply:
Change to BeamIOError will miss generic IOError (though BeamIOError extends from IOError). For example, for local filesystem, an IOError is raised:

https://github.com/apache/beam/blob/1d3ba8c6db91c0e781ccab60d02f1fbc243dd6e8/sdks/python/apache_beam/io/localfilesystem.py#L78-L81

I just copy-pasted the same code in https://github.com/apache/beam/blob/1d3ba8c6db91c0e781ccab60d02f1fbc243dd6e8/sdks/python/apache_beam/io/fileio.py#L653

yeah we can output a log if it is not `FileExistsError`